### PR TITLE
Distinct instanceDir and wd

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
Fix the issue that `tiup_process_meta` is generated at the wd and `tiup status` can not work.

### What is changed and how it works?
After #682, we do not change wd for the run components.
It make it save the tiup_process_meta file in wd instead of
InstanceDataDir like before. This seems broke "tiup clean" and "tiup
status" which check all subdirectory in ~/.tiup/data for the
tiup_process_meta.

After this PR, we Distinct InstanceDataDir and wd in launchComponent and
make Dir in tiup_process_meta as the InstanceDataDir instead of wd.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
run `tiup tidb` and check `tiup status`

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/exec/run.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func PrepareCommand(
 	ctx context.Context,
 	component string,
 	version v0manifest.Version,
-	binPath, tag, wd string,
+	binPath, tag, instanceDir string, wd string,
```
Comment (lonng) [inline pkg/exec/run.go:138]: Seems we don't need `wd` anymore.

Comment (july2993) [reply pkg/exec/run.go:138]: playground is using this `PrepareCommand` too and set the wd as ~/.data/{playground}/{instance}

**Review by lonng — APPROVED** [review]
LGTM

**Pull request merged**